server/gokbg3/grails-app/controllers/gokbg3/UrlMappings.groovy

```diff
@@ -16,6 +16,7 @@ class UrlMappings {
     group "/rest", {
       post "/packages/$id/retire"(controller: 'package', namespace: 'rest', action:'retire')
       post "/packages/$id/tipps"(controller: 'package', namespace: 'rest', action: 'addTipps')
+      put "/packages/$id/tipps"(controller: 'package', namespace: 'rest', action: 'updateTipps')
       "/packages/$id/$action"(controller: 'package', namespace: 'rest')
```

Comment:
just add `patch "/packages/$id/tipps"(controller: 'package', namespace: 'rest', action: 'updateTipps')` as well

Reply:
Agreed. PATCH now has the behaviour of `addOnly` by default, where missing TIPPs are simply ignored.